SparkPost/2web2ui
# SA-1616: Compare By stores in query params

### What Changed
 - Added submission for compare by to modify state
 - Updates in query params and preserves in query params

### How To Test
 - Enable `allow_compare_by` feature flag
 - Fill out form
 - Submit
 - Verify that the query params properly reflect the compare filters
 - Refresh page
 - Verify that the form properly populates with the appropriate state

### To Do
- [ ] Additional testing
